src/api/artifacts-api.js

```diff
@@ -65,6 +65,16 @@ const artifactsApi = {
 
     return mainHttpClientV2.delete(`/projects/${project}/artifacts/${key}`, config)
   },
+  deleteArtifacts: (project, name, category) => {
+    const config = {
+      params: {}
+    }
+    
+    if (name) config.params.name = name
+    if (category) config.params.category = category
```

Comment:
category param doesn't works for now, I've added it for future, when on BE this param will be added